requirements-test.txt

```
@@ -3,3 +3,4 @@ pytest==4.6.5
 pytest-cov==2.5.1
 pytest-pep8>=1.0.6,<2.0.0
 coveralls>=1.8.2,<2.0.0
+requests_mock
```

As mentioned elsewhere, I'm a big fan of `pytest-mock` and the `mocker` fixture.

```suggestion
requests_mock
pytest-mock
```